.goreleaser.yaml

```diff
@@ -0,0 +1,113 @@
+project_name: rill
+
+release:
+  prerelease: auto
+  draft: true
+
+builds:
+  - id: darwin_amd4
+    main: ./cli/main.go
+    binary: rill
+    goos:
+      - darwin
+    goarch:
+      - amd64
+    env:
+      - PKG_CONFIG_SYSROOT_DIR=/sysroot/macos/amd64
+      - PKG_CONFIG_PATH=/sysroot/macos/amd64/usr/local/lib/pkgconfig
+      - CC=o64-clang
+      - CXX=o64-clang++
+    flags:
+      - "-mod=readonly"
+    ldflags:
+      - -s -w -X main.Version={{.Version}} -X main.Commit={{.ShortCommit}} -X main.BuildDate={{.Date}}
+
+  - id: darwin_arm64
+    main: ./cli/main.go
+    binary: rill
+    goos:
+      - darwin
+    goarch:
+      - arm64
+    env:
+      - PKG_CONFIG_SYSROOT_DIR=/sysroot/macos/amd64
+      - PKG_CONFIG_PATH=/sysroot/macos/amd64/usr/local/lib/pkgconfig
+      - CC=oa64-clang
+      - CXX=oa64-clang++
+    flags:
+      - "-mod=readonly"
+    ldflags:
+      - -s -w -X main.Version={{.Version}} -X main.Commit={{.ShortCommit}} -X main.BuildDate={{.Date}}
+
+  - id: linux
+    main: ./cli/main.go
+    binary: rill
+    goos:
+      - linux
+    goarch:
+      - amd64
+    env:
+      - CC=x86_64-linux-gnu-gcc
+      - CXX=x86_64-linux-gnu-g++
+      - CGO_CFLAGS=--sysroot=/sysroot/linux/amd64
+      - CGO_LDFLAGS=--sysroot=/sysroot/linux/amd64
+      - PKG_CONFIG_SYSROOT_DIR=/sysroot/linux/amd64
+      - PKG_CONFIG_PATH=/sysroot/linux/amd64/usr/local/lib/pkgconfig
+    flags:
+      - "-mod=readonly"
+    ldflags:
+      - -s -w -X main.Version={{.Version}} -X main.Commit={{.ShortCommit}} -X main.BuildDate={{.Date}}
+
+blobs:
+  - provider: gs
+    bucket: stage-cdn.rilldata.io #TODO: testing
+    folder: "rill/v{{ .Version }}"
+    extra_files:
+      - glob: ./dist/**/*
+
+  - provider: gs
+    bucket: stage-cdn.rilldata.io #TODO: testing
+    folder: "rill/latest"
+    extra_files:
+      - glob: ./dist/**/*
+
+brews:
+  - homepage: https://github.com/rilldata/rill-developer
+    description: The Rill CLI
+    name: rill
+    license: Apache 2.0
+    url_template: https://cdn.rilldata.io/rill/{{ .Tag }}/{{ .ArtifactName }} #TODO: testing
+    tap:
+      owner: rilldata
+      name: homebrew-rill-developer
+      branch: main
+      token: "{{ .Env.GITHUB_TOKEN }}"
+    test: |
+      system "#{bin}/rill --version"
+    install: |
+      bin.install "rill"
+    commit_author:
+      name: rilldata
+      email: support@rilldata.com
+
+archives:
+  - replacements:
+      darwin: Darwin
+      linux: Linux
+    files:
+      - README.md
+      - LICENSE*
+    name_template: "{{ .ProjectName }}_{{ .Os }}_{{ .Arch }}"
+
+checksum:
+  name_template: checksums.txt
+
+snapshot:
+  name_template: "{{ incpatch .Version }}-snapshot"
+
+changelog:
+  sort: asc
+  filters:
+    exclude:
+      - "^docs:"
+      - "^test:"
```

Comment:
Do these excludes match commit messages? We don't use prefixes for commit messages like this, so they'll have no effect. Is there a way to match paths instead?

Reply:
Yeah as per their documentation we have 

for e.g:

`# Paths to filter the commits for.
  # Only works when `use: git`, otherwise ignored.
  # Only on GoReleaser Pro.
  #
  # Default: monorepo.dir value, or empty if no monorepo.
  # Since: v1.12.0-pro
  paths:
  - foo/
  - bar/
`

Reply:
For now removing filters as its of no use